tests/baselines/reference/elidedJSImport1.errors.txt

```diff
@@ -0,0 +1,31 @@
+tests/cases/compiler/caller.js(1,21): error TS2307: Cannot find module 'fs' or its corresponding type declarations.
+tests/cases/compiler/caller.js(2,8): error TS18042: 'TruffleContract' is a type and cannot be imported in JavaScript files. Use 'import("@truffle/contract").TruffleContract' in a JSDoc type annotation.
+tests/cases/compiler/caller.js(4,43): error TS2708: Cannot use namespace 'TruffleContract' as a value.
+tests/cases/compiler/caller.js(4,60): error TS2708: Cannot use namespace 'TruffleContract' as a value.
+
+
+==== tests/cases/compiler/caller.js (4 errors) ====
+    import * as fs from 'fs';
+                        ~~~~
+!!! error TS2307: Cannot find module 'fs' or its corresponding type declarations.
+    import TruffleContract from '@truffle/contract'; // Runtime err: this import is elided in transform
+           ~~~~~~~~~~~~~~~
+!!! error TS18042: 'TruffleContract' is a type and cannot be imported in JavaScript files. Use 'import("@truffle/contract").TruffleContract' in a JSDoc type annotation.

```

Comment:
Note that if `checkJs` is `true`, the user will still get the errors because we believe `TruffleContract` is a type and not a value.